Added note about "Used in" functionallity (#3)
AdobeDocs/commerce-admin.en: help/content-design/media-gallery-asset-management.md

## Filtering assets

>[!NOTE]
>

> >To use "Used in" functionallity "Media Gallery Image Optimization" should be enabled in _Admin_ sidebar **[!UICONTROL Stores]** > _[!UICONTROL Configuration]_ > **[!UICONTROL Advanced]** > _[!UICONTROL System]_ > **[!UICONTROL Media Gallery Image Optimization]**

Reviewer: Simplify and link to the configuration information.

```suggestion
>The _Used in_ functionality requires that [!UICONTROL Media Gallery Image Optimization] is enabled in the [configuration settings](media-gallery-image-optimization.md).
```
